app/src/main/res/layout/promoted_story_card.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can you confirm as part of #134 with Chantel whether this layout matches the case where we show a suggested lesson instead of a recently completed? I'm guessing it does, but I'm having difficulty tracking that down in the mocks.

I don't think this question should block this PR, but it's good to clarify.

Reply:
Sent an email.